capstone/src/main/java/com/google/univiz/Grapher.java

```diff
@@ -0,0 +1,22 @@
+package com.google.univiz;
+
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import com.google.inject.grapher.graphviz.GraphvizGrapher;
+import com.google.inject.grapher.graphviz.GraphvizModule;
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class Grapher {
+  public static void main(String... args) throws IOException {
+    String filename = "guice_graph.dot";
+    PrintWriter out = new PrintWriter(new File(filename), "UTF-8");
+    Injector demoInjector = Guice.createInjector(new UnivizBootstrapModule());
+    Injector injector = Guice.createInjector(new GraphvizModule());
+    GraphvizGrapher grapher = injector.getInstance(GraphvizGrapher.class);
+    grapher.setOut(out);
+    grapher.setRankdir("TB");
```

Comment:
nit - i think you can delete this since it's [the default](https://github.com/google/guice/blob/master/extensions/grapher/src/com/google/inject/grapher/graphviz/GraphvizGrapher.java#L56).